web/src/components/molecules/Asset/Asset/AssetBody/Asset.tsx

```diff
@@ -119,7 +119,7 @@ const AssetMolecule: React.FC<Props> = ({
             />
           </Card>
         )}
-        <DownloadButton type="ghost" filename={asset.fileName} url={asset.url} displayDefaultIcon />
+        <DownloadButton type="ghost" selected={asset ? [asset] : undefined} displayDefaultIcon />
```

Comment:
I prefer to have this check inside the atom, so it will give a higher flexibility, to either pass an array or a single file.

Reply:
I'll leave the refactoring/removing/etc of the Download button up to you, okay? It isn't a great component right now and again would make more sense as a molecule or to be removed completely since it is just a glorified Antd Button.

Reply:
@KaWaite , ok, sure.

Reply:
I'll resolve this for now.